build/makefile

```diff
@@ -13,7 +13,10 @@ ifeq ($(OS),Windows_NT)
 	PLATFORM_OPTS = -static
 	TBB_PLATFORM_OPTS = -DUSE_WINTHREAD
 else
-	LD_PLATFORM_OPTS = -lrt
+	UNAME_S := $(shell uname -s)
```

Comment:
Why not use `$(OS)`?

Reply:
Unfortunately doesn't work on OSX at least. I based this design off of this pattern:

https://stackoverflow.com/questions/714100/os-detecting-makefile